values.yaml

```diff
@@ -77,9 +77,13 @@ dfclient:
   ## Additional dfclient hostPath mounts
   ##
   extraHostPathMounts: []
-    # - name: logs
-    #   mountPath: /root/.small-dragonfly/logs
-    #   hostPath: /var/dragonfly/logs
```

Comment:
For me, this hostpath volume can help collect these dfclient logs by log agent. Is it better to keep it?

Reply:
Agree. I will change that.

Reply:
Done.